extension/keysocket-hypem.js

```diff
@@ -1,9 +1,9 @@
 function onKeyPress(key) {
-    if(key === NEXT) {
-        simulateClick(document.querySelector('#playerNext'));
-    } else if(key === PLAY) {
-        simulateClick(document.querySelector('#playerPlay'));
-    } else if(key === PREV) {
-        simulateClick(document.querySelector('#playerPrev'));
-    }
+	if(key === NEXT) {
```

Comment:
Yeah tabs, my editor converted, sorry I will fix them.

To confirm the preferred indentation is 4 spaces, yes?